modules/javafx.graphics/src/main/docs/javafx/scene/doc-files/cssref.html

```diff
@@ -742,6 +742,16 @@ <h3><a id="introtransitions">Transitions</a></h3>
         </tr>
       </tbody>
     </table>
+    <p>A transition is not started (or cancelled if already running) in any of these scenarios:
+      <ol>
+        <li>The property value is set programmatically</li>
+        <li>The property is bound</li>
+        <li>The node becomes invisible</li>
```

Comment:
I would mention that this relates to the `visible` property and not to the `opacity` one (the node is invisible if opacity is 0).

Other than that, looks good.

Reply:
I've changed it to `The node (or any of its parents) is invisible, as indicated by the Node.visible property`.